src/components/Tabs/Tabs.tsx

```diff
@@ -65,6 +74,110 @@ const TabsComponent = ({
 
   const withGaps = mode === "accent" || mode === "secondary";
 
+  function getTabEls(): HTMLDivElement[] {
```

Comment:
```suggestion
  const getTabEls = (): HTMLDivElement[] => {
```

заменишь плиз на стрелочную функцию 🙏